pkg/storaged/lvol-tabs.jsx

```diff
@@ -185,6 +197,13 @@ function get_resize_info(client, block, to_fit) {
                     grow_excuse = cockpit.format(_("$0 filesystems can not be made larger."),
                                                  block.IdType);
             }
+        } else if (client.blocks_stratis_blockdev[block.path]) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-120835-5b4a3bfc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)